server/graphql/v2/query/collection/ExpensesCollectionQuery.ts

```diff
@@ -375,6 +375,32 @@ const ExpensesCollectionQuery = {
     return {
       nodes: result.rows,
       totalCount: result.count,
+      totalAmount: async ({ currency = 'USD' }) => {
+        const totals = (await models.Expense.findAll({
+          attributes: [
+            [Sequelize.col('"Expense"."currency"'), 'expenseCurrency'],
+            [Sequelize.fn('SUM', Sequelize.col('amount')), 'amount'],
+          ],
+          group: 'expenseCurrency',
+          include,
+          where,
+          raw: true,
+        })) as unknown as { expenseCurrency: string; amount: number }[];
+
+        const currencies = totals.map(result => ({ currency: result.expenseCurrency, value: result.amount }));
+
+        const values = await req.loaders.CurrencyExchangeRate.convert.loadMany(
+          currencies.map(v => ({ amount: v.value, fromCurrency: v.currency, toCurrency: currency })),
+        );
+
+        return {
+          currencies,
+          converted: {
+            value: sum(values),
+            currency,
+          },
```

Comment:
@Betree I think the schema is still a bit awkward, if you have suggestions let me know. 
I tried to add flexibility here by aggregating the total by currency and returning each value for the frontend to display. Optionally, a desired currency can be passed as an argument to convert everything.

In some uses of the expense query, the desired currency is not clear:
- If we querying expenses submitted to different collectives/hosts.
- If we have a mix of PAID/UNPAID expenses, converting to a single currency might not make sense with different exchange rates across time.